content.php

```diff
@@ -11,39 +11,60 @@
     
     if ( $imageAttributes ) {
         $articleStyle = 'style="background-image: url( ' . $imageAttributes[0] . ' )"';
```

Comment:
`$articleStyle` isn't used anymore. Read through lines 6-29, the code here can be simplified by a lot since there are some code that does nothing.